### PR TITLE
Report INVALID_ENUM for ETC2/EAC formats without WEBGL_compressed_texture_etc

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

File: sdk/tests/conformance/extensions/webgl-compressed-texture-etc.html [hunk 1]
```diff
@@ -140,11 +140,12 @@
 
     debug("");
     if (contextVersion >= 2) {
+      var expectedError = isPositive ? gl.INVALID_OPERATION: [gl.INVALID_ENUM, gl.INVALID_OPERATION];
```
Comment (kainino0x) [inline sdk/tests/conformance/extensions/webgl-compressed-texture-etc.html]: It seems a little odd to allow two either INVALID_ENUM or INVALID_OPERATION. But the difference may not matter enough to specify and fix.

**Comment by kainino0x** [comment]
@qkmiao thanks for this fix. I added one comment for discussion.

**Comment by kenrussell** [comment]
It looks like these calls can generate one or the other error, which is why both have to be allowed for WebGL 2.0.

Thanks @qkmiao for this fix. Merging now.

File: sdk/tests/conformance/extensions/webgl-compressed-texture-etc.html (hunk 1 above)
Comment (qkmiao) [inline sdk/tests/conformance/extensions/webgl-compressed-texture-etc.html]: We don't know which error will be raised early by implementations, either INVALID_ENUM or INVALID_OPERATION is possible in this case.

**Pull request merged**